README.md

```diff
@@ -1 +1,216 @@
-# Doctor-Reservation-backend
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Doctors Reservation Backend</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Doctors Reservation Backend ](#-Doctors-Reservation-Backend-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Setup database](#setup-database)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Doctors Reservation Backend <a name="about-project"></a>
+
+**Doctors Reservation Backend** is a Rails API that serves as database to the [Doctors Reservation Frontend](https://github.com/Iqbal-Elham/Doctor-Reservation-frontend) project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Create database**
+- **Add a doctor for reservation**
+- **Local storage functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Render](*https://*)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+  ruby 3.2.1
+  rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd your-folder
+  git clone git@github.com/Iqbal-Elham/Doctor-Reservation-backend.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Setup database
+
+Setup database with:
+
+Example command:
+
+```sh
+  rails db:create
+  rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  rspec
+```
```

Comment:
- [ ] When I run the tests according to the instructions, this happens:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/0cee5141-80a3-467d-9869-df2eef104df9)

This is because all your tests are rswag tests.  Rspec won't run them correctly.

The command that will run them correctly is `rake rswag`:

![image](https://github.com/Iqbal-Elham/Doctor-Reservation-backend/assets/57421823/b3648839-fea8-466e-8b88-9c41c5b94b6a)

Kindly, fix your instructions so your tests pass. 👍 

_note: This command is idempotent, which means it is safe to run every time and will yield the same result, no matter how many times you or your users run it._